src/DataTable/Table.jsx

```diff
@@ -3,24 +3,33 @@ import PropTypes from 'prop-types';
 import classNames from 'classnames';
 import TableHeaderRow from './TableHeaderRow';
 import TableRow from './TableRow';
+import { useRows } from './hooks';
 
 const Table = ({
-  getTableProps, headerGroups, getTableBodyProps, rows, prepareRow, isStriped,
+  isStriped, ...rest
```

Comment:
An aside: I've used `...attrs` for allowing passthroughs to html elements and `...props` for passthroughts to components. I've seen `...rest` like you have here. Do you have thoughts on a consistent guide for this?

Reply:
Yeah, I've seen all kinds of things. I'm happy to go with `...props` so it's consistent across this repo at least.